api/libproject/devstart.rb

```diff
@@ -930,6 +930,8 @@ def import_cdr_indices_build_to_cloudsql(cmd_name, *args)
 
   ENV.update(read_db_vars(gcc))
   ENV.update(must_get_env_value(gcc.project, :gae_vars))
+  ENV["DB_HOST"] = "127.0.0.1" # TODO(dmohs): Use MYSQL_TCP_PORT to be consistent with mysql CLI.
```

Comment:
You mean MY_SQL_HOST?

Reply:
Those comments are old.. i will remove them.